conf/credentials.yaml.template

```diff
@@ -32,6 +32,7 @@ netapp-storage:
 database:
     username: root
     password: password
+    port: 5432
```

Comment:
I'm not sure about this. I'd prefer to keep credentials just have credential related things in it, @mfalesni your opinion?